assets/src/styles/_theme-palette-onondaga.scss

```diff
@@ -0,0 +1,32 @@
+// USWDS Project theme palette
```

Comment:
Is this a theme that will eventually be available via USWDS? I'm having difficulty finding any information about this so I'm confused as to what this is doing, so if you have documentation link that would be helpful.

Reply:
Yes unfortunately the documentation is poor currently, and we'll need to visit this when it's updated. But this theme was copy-pasted from here: https://federalist-proxy.app.cloud.gov/preview/gsa/10x/dw-design-refinements/uswds-color-docs-theme/#0 I chose that one because it's the only one I could find - the download links on the others don't work.

Reply:
Thanks for the link. I thought I had bookmarked that but couldn't find it. The nice thing about using themes is it will be easy to change when ACWI. I was thinking the Cascade matches their theme colors but it more closely matches the National Water Quality Monitoring Council (a ACWI group).